app/lib/analytics.coffee

```diff
@@ -74,9 +74,9 @@ logToGoogle = (eventData) =>
 ###
 This will log a user interaction both in the Geordi analytics API and in Google Analytics.
 ###
-logEvent = (type, related_id = '', user_id = User.current?.zooniverse_id, subject_id = ExperimentalSubject.current?.zooniverseId) =>
+logEvent = (type, related_id = '', subject_id = ExperimentalSubject.current?.zooniverseId) =>
```

Comment:
Agreed. I already have this on my to-do list, I plan to address it as part of https://github.com/zooniverse/geordi/issues/12